app/src/androidTest/java/ch/epfl/swissteam/services/SettingsFragmentTest.java

```diff
@@ -55,6 +56,39 @@ public void canOpenSettingsFragment() {
         onView(withId(R.id.nav_view)).perform(NavigationViewActions.navigateTo(R.id.button_maindrawer_settings));
     }
 
+    @Test
+    public void setHomeCanBePressed() {
+        onView(withId(R.id.drawer_layout)).perform(DrawerActions.open());
+        onView(withId(R.id.nav_view)).perform(NavigationViewActions.navigateTo(R.id.button_maindrawer_settings));
+        sleep(100);
+        closeSoftKeyboard();
+        sleep(500);
+        onView(withId(R.id.button_settings_sethome)).perform(scrollTo()).perform(click());
+    }
+
+/*
+    @Test
+    public void setHomeToCurrentLocationWorks() {
+        onView(withId(R.id.drawer_layout)).perform(DrawerActions.open());
+        onView(withId(R.id.nav_view)).perform(NavigationViewActions.navigateTo(R.id.button_maindrawer_settings));
+
+        onView(withId(R.id.button_settings_sethome)).perform(click());
+        double lat = ((SettingsFragment) mActivityRule.getActivity()).getHomeLat_();
+        double lng = (SettingsFragment) mActivityRule.getActivity().getView()
+        assertEquals().;
+    }
+*/
+/*
+    @Test
+    public void isHomeMarkerMatchHomeLocation() {
+        onView(withId(R.id.drawer_layout)).perform(DrawerActions.open());
+        onView(withId(R.id.nav_view)).perform(NavigationViewActions.navigateTo(R.id.button_maindrawer_settings));
+
+        onView(withId(R.id.button_settings_sethome)).perform(click());
+
```

Comment:
I'm not sure that it is testable :disappointed:

Reply:
Thanks for all your advice :)